lms/djangoapps/course_api/blocks/tests/test_api.py

```diff
@@ -4,8 +4,12 @@
 
```

Comment:
**Note to reviewers:** `lms/djangoapps/course_api/blocks/tests/test_api.py` was updated so its integration tests would test block-structures with and without storage backing.